pkg/deploytest/helpers.go

```diff
@@ -30,5 +30,5 @@ func NewLogger(parentLogger logging.Logger) logging.Logger {
 	if parentLogger != nil {
 		return logging.Synchronize(parentLogger)
 	}
-	return logging.Synchronize(logging.ConsoleDebugLogger)
+	return logging.ConsoleDebugLogger // No need to synchronize, the ConsoleDebugLogger is already synchronized.
```

Comment:
Correct, but the synchronize function already checks this and exits quickly if that's the case. I do not see a scenario where the logger won't be synchronized already here though.

Reply:
(feel free to resolve)

Reply:
Yes that is right. And if we were using an arbitrary logger (that could be synchronized or not synchronized) I'd absolutely leave it there. But since this is a concrete logger of which we know is always synchronized, synchronizing it again is confusing and makes the code harder to understand.